site/content/entry/yuzu-mini-dev-3/index.md

```diff
@@ -0,0 +1,91 @@
+If you didn't know already, NVDEC is Nvidia's video decoder used by the Nintendo Switch. Without him we wouldn't be able to watch videos like Fire Emblem: Three House's intro or The Legend of Zelda: Breath of the Wild's memories.
+But that's not the only thing he has done! 
+He was also responsible for asynchronous shader compilation on the Vulkan renderer, GameCube Controller support and [Alpha Test Culling on the Vulkan renderer](https://github.com/yuzu-emu/yuzu/pull/4946).
+Let's jump right into the interview!
+***
+
+**Q: Hey epicboy, why don't you start by telling us a bit about yourself?**
+
+**E:** I'm just a typical college student majoring in Computer Science. I'm in my fourth year of studies out of five, pursuing both a BS and MS degree. 
+To be honest, I've been suffering from imposter syndrome in my classes, so I wanted to put my programming skills to the test. 
+That's what drove me to contribute to yuzu; a real-world test of the knowledge and experience I gained from my classes to prove to myself that my time at college hasn't been a waste.
+
+**Q: Why did you choose yuzu, a Nintendo Switch emulator, specifically?**
+
+**E:** When I first heard of yuzu, back in 2018, I was impressed by the progress made on Switch emulation so soon after the console's release. 
+I continued to follow the project's progress and was inspired by the never ending drive and effort of the developers. 
+Soon after SSBU was announced as playable on the emulator, which sounded too good to be true, I gave the emulator a shot and was blown away at the compatibility. 
+Of course, it wasn't perfect, especially given that my beloved controller of choice, the Nintendo GameCube controller, was not compatible with the emulator. 
+
+**Q: So you decided that it was time to take matters into your own hands and add support yourself?**
+
+**E:** Knowing that other emulators, notably [Dolphin](https://dolphin-emu.org/) had proper support for the official GameCube adapter, 
+and considering I was looking to sharpen my programming skillset, 
+I decided to take the opportunity and see if I can bring that support over to yuzu, opening the option to use the GameCube controller for anyone on the emulator.
+
+**Q: I'm sure our GC controller users are still very happy with your contribution! Why did you choose to tackle such an grand task as NVDEC?**
+
+**E:** I found the code review for the GC adapter implementation to be really useful. 
+I wanted to learn more from the very sharp yuzu developers and continued to make more contributions and became more familiar with the yuzu dev team. 
+One feature that I saw many yuzu fans request was for video decoding. 
+I learned that [ogniK](https://github.com/ogniK5377) had attempted at implementing nvdec before and reached out to get an understanding of this feature's scope, and challenges faced along the way. 
```

Comment:
Change `had attempted at implementing nvdec` to `had attempted to implement nvdec`